Tests/BraveWalletTests/SelectAccountTokenStoreTests.swift

```diff
@@ -133,14 +157,22 @@ import Preferences
           accountInfos: [.mockSolAccount]
         )
         completion(keyring)
+      } else if keyringId == BraveWallet.KeyringId.filecoin {
+        completion(.mockFilecoinKeyringInfo)
+      } else {
+        completion(.mockFilecoinTestnetKeyringInfo)
```

Comment:
Probably better to use a `switch` now that there are 4 different cases just comparing strings.